zds/member/views.py

```diff
@@ -120,6 +120,11 @@ def get_form(self, form_class=ProfileForm):
     def post(self, request, *args, **kwargs):
         form = self.form_class(request.POST)
 
+        if 'preview' in request.POST:
+            if request.is_ajax():
```

Comment:
serait-il possible de mettre `if 'preview' in request.POST and request.is_ajax()` afin de limiter le nombre de branches et la profondeur de l'indentation?